encrypted-parameters.md

```diff
@@ -0,0 +1,50 @@
+---
+---
+
+# Encrypted Input Parameters
+
+The parameters passed to a playbook method are often encrypted, either by definition as a “password” data type in the list of input parameters, or by being encrypted elsewhere in a workflow.
+
+## Encrypted Input Parameters
+
+An input parameter can be defined as being of type “password”, for example:
+
+Input Parameters
+
+|                           |                           |                              |
+| ------------------------- | ------------------------- |----------------------------- |
+| **Input Name**            | **Data Type**             | **Default value**            |
+| this                      | string                    | that                         |
```

Comment:
```suggestion
| **Input Name**            | **Data Type**             | **Default value**            |
| ------------------------- | ------------------------- |----------------------------- |
| this                      | string                    | that                         |
```

Reply:
The formatting looks strange, but the line before the `| ------------------------- |` is the header, so the header lines should live there.  Note that you don't need to bold it because renderers will do that automatically.